test/test_unary_ufuncs.py

```diff
@@ -89,14 +96,18 @@ def generate_tensors_from_vals(vals, device, dtype, domain):
         if domain[1] is not None:
             vals = list(filter(lambda x: x < domain[1], vals))
 
+    condition, safe_value = filter_
+
     # Constructs the large tensor containing vals
     large_tensor = make_tensor(_large_size, device=device, dtype=dtype, low=domain[0], high=domain[1])
 
     # Inserts the vals at an odd place
     large_tensor[57][offset:offset + len(vals)] = torch.tensor(vals, device=device, dtype=dtype)
+    replace_values_in_tensor(large_tensor, condition, safe_value)
```

Comment:
Let's guard these calls on `condition` not being None so we don't invoke extra kernels in the common case where `condition` is None